components/dashboard/src/Menu.tsx

```diff
@@ -268,25 +264,39 @@ export default function Menu() {
         setFeedbackFormVisible(false);
     };
     const renderTeamMenu = () => {
+        const classes =
+            "flex h-full text-base py-0 " +
+            (!projectSlug && !isWorkspacesUI && !isAdminUI && teamOrUserSlug
+                ? "text-gray-50 bg-gray-800 dark:text-gray-900 dark:bg-gray-50 border-gray-700"
+                : "text-gray-500 bg-gray-50 dark:bg-gray-800 dark:text-gray-400 hover:bg-gray-100 dark:hover:bg-gray-700  dark:border-gray-700");
```

Comment:
**thought**: Probably not the best way to handle classes for the left and right part of the team scope selector. 😇